mk/aosp_optee.mk: Allow extra flags for aosp builds

vchong opened the pull request:
Set core and TA log level to 3.
Set debug build.

Signed-off-by: Victor Chong <victor.chong@linaro.org>

jenswi-linaro commented on mk/aosp_optee.mk:
```diff
@@ -50,7 +50,10 @@ BUILD_OPTEE_OS:
 		PLATFORM_FLAVOR=$(OPTEE_PLATFORM_FLAVOR) \
-		$(CROSS_COMPILE_LINE)
+		$(CROSS_COMPILE_LINE) \
+		CFG_TEE_CORE_LOG_LEVEL=3 \
```
Shouldn't these be configurable?

vchong replied:
Yes, but people who use these builds usually want to develop their own TAs, so the default setting should be friendly to them, i.e. debuggable and logs viewable. Otherwise they'll just end up having to rebuild again. E.g. #1438 above.

vchong replied:
Or can we add something like `$(OPTEE_EXTRA_FLAGS)` here and try to set the log level from one of the aosp .mk files?

jenswi-linaro replied:
Yes, that's fine also.

vchong commented:
Updated. Title changed to `mk/aosp_optee.mk: Allow extra flags for aosp builds`. Will update commit message when adding tag(s).

vchong commented:
Sorry. Closed by accident.

jenswi-linaro commented:
+1 for testing that the close button still works. :-P
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

vchong commented:
Yes, the lack of early morning caffeine can sometimes trigger extra random button testing automatically! ;)

Squashed, commit message updated and tag added. Rebased not required. Ok to merge. Thanks!

jforissier commented:
Rebased on top of master and merged. Thanks!
